src/helix-ui/styles/components/files/hx-file-control.less

```diff
@@ -0,0 +1,78 @@
+// ----- LAYOUT -----
+hx-file-control {
+  > * {
+    margin: 0;
+  }
+
+  > label {
+    display: inline-block;
+    padding: 0;
+  }
+
+  > input[type="file"] {
+    #Mix.hideInput();
+  }
```

Comment:
Need to add the following CSS to remove the "dot" that appears in Edge and IE...

```less
hx-file-control > input[type="file"] {
  background: none;
}
hx-file-control > input[type="file"]::-ms-browse,
hx-file-control > input[type="file"]::-ms-value { 
  visibility: hidden; 
}
```